packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -163,17 +167,20 @@ function _getDomBreadcrumbHandler(
       return;
     }
 
-    addBreadcrumb(
-      {
-        category: `ui.${handlerData.name}`,
-        message: target,
-      },
-      {
-        event: handlerData.event,
-        name: handlerData.name,
-        global: handlerData.global,
-      },
-    );
+    const breadcrumb: Breadcrumb = {
+      category: `ui.${handlerData.name}`,
+      message: target,
+    };
+
+    if (componentName) {
+      breadcrumb.data = { componentName };
```

Comment:
let's call this `ui.component_name` and align it with the other setting of data

Reply:
Will do 👍